SwiftWisdom/Core/StandardLibrary/Array/Sequence+Utilities.swift

```diff
@@ -54,6 +54,7 @@ extension Sequence where Iterator.Element: Equatable {
     /// Provides only the unique values within a `Sequence`. ie: "1, 2, 2, 3" will return "1, 2, 3".
     ///
     /// - returns: Returns a sequence of the unique values
+    // TODO: Refactor to use Set for buffer?
```

Comment:
Not sure we want to do that. While this implementation is slow, it does maintain order, which I view as a benefit. It might be better just to document that feature of the function.

Reply:
Excellent point. I've added to the documentation.

Reply:
Though it would be pretty easy to maintain order and O(n) time complexity:
```
var buffer = [Element]()
var seen = Set<Element>()
for element in self where !seen.contains(element) {
  buffer.append(element)
  seen.insert(element)
}
return buffer
```

Reply:
Love it.